src/lodashMin.ts

```diff
@@ -0,0 +1,36 @@
+/**
+* lodashMini.ts
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Imports a subset of lodash library needed for SimpleRestClient
+*/
+
+import clone = require('lodash.clone');
```

Comment:
Please use lodash/module name instead of lodash.module name to stay consistent with our other projects
(https://github.com/Microsoft/reactxp/blob/master/src/common/lodashMini.ts as reference)